test/metabase/api/search_test.clj

```diff
@@ -28,25 +29,38 @@
    [schema.core :as s]
    [toucan.db :as db]))
 
+(defn- ordered-subset?
+  "Test if all the elements in xs appear--in order--in ys. Search results in this test suite can be polluted by local
+  data, so this is a way to ignore extraneous results."
+  [[x & rest-x :as xs] [y & rest-y :as ys]]
+  (if (zero? (count xs))
+    true
+    (if (> (count xs) (count ys))
+      false
+      (if (= x y)
+        (recur rest-x rest-y)
+        (recur xs rest-y)))))
```

Comment:
I frequently get bitten by my local DB state when I run these tests locally. So I rewrote a few things to mitigate that. I don't think we're 100% there yet, but it's at least better (and working on my machine! :wink: )